.github/workflows/ci-checks-build.yml

```diff
@@ -60,7 +60,6 @@ jobs:
           path: |
             # /var/cache/apt/archives/*.deb
             ~/.m2/repository
-            /opt/intel/oneapi
             ~/opt
           key: ${{ runner.os }}_spark-3.2.0_hadoop-3.2.0_oneapi-2021.4.0
```

Comment:
you don't need to remove /opt/intel/oneapi, just need to change key name(${{ runner.os }}_spark-3.2.0_hadoop-3.2.0_oneapi-2021.4.0), same below

Reply:
Done.